dev/test/util.ts

```diff
@@ -58,51 +58,50 @@ describe('isPlainObject()', () => {
 
     it('reads true', async () => {
       process.env.FIRESTORE_PREFER_REST = 'true';
-      const [prevSet, prevValue] = tryGetPreferRestEnvironmentVariable();
-      expect(prevSet).to.be.true;
+      const prevValue = tryGetPreferRestEnvironmentVariable();
```

Comment:
nit: Can you rename `prevValue` to avoid the abbreviation? I don't know what "prev" is an abbreviation for. Is it "previous"? Oh, is it an acronym for "prefer rest environment variable"? Consider a name like "returnValue", "result" or "actual", or even just "value". Even better, get rid of the local variable and merge it into the next line, like this:

```ts
expect(tryGetPreferRestEnvironmentVariable()).to.be.true;
```

I actually find that to be the most readable because it "reads like a story" from left to right.

In general, non-standard abbreviations detract from readability. Avoid them.

Applies here and elsewhere in this file.